tests/test_lecker_2.py

```diff
@@ -0,0 +1,38 @@
+from recipe_scrapers.lecker import Lecker
+from tests import ScraperTest
+
+
+class TestLeckerScraper(ScraperTest):
+
+    scraper_class = Lecker
+
+    @property
+    def test_file_name(self):
+        return "{}_2".format(self.scraper_class.__name__.lower())
+
+    def test_host(self):
+        self.assertEqual("lecker.de", self.harvester_class.host())
+
+    def test_title(self):
+        self.assertEqual(
+            "Naan-Brot backen - so geht das Rezept",
+            self.harvester_class.title(),
+        )
+
+    def test_image(self):
+        self.assertEqual(
+            "https://www.lecker.de/assets/field/image/naan-brot-b_0.jpg",
+            self.harvester_class.image(),
+        )
+
+    def test_ingredients(self):
+        self.assertEqual(
+            [],
+            self.harvester_class.ingredients(),
+        )
+
+    def test_instructions(self):
+        self.assertIn(
+            "Als Naan-Brot bezeichnet man dünne Teigfladen aus m",
```

Comment:
Could you update this test to contain more of the instructions?

(it's helpful to provide visibility that the scraper is extracting the correct content)

Reply:
i found it very ugly to contain the whole ingredient string in the test file but if you prefer it that way sure, i will change it, also for the other two PRs were I used assertIn

Reply:
Yep, that'd be great, thanks - it helps when reviewing the code because it's a reassurance that the scraper does retrieve the appropriate text, and isn't missing/including other details.

Reply:
stupid question but how do you normally get the string of the instructions. 

If I pause the execution to copy the value of `scraper.instructions()` i get a string containing the line breaks as `\n` but if i paste it in pycharm or notepadd++ the `\n`'s are always converted into actual line breaks.

Reply:
There aren't many stupid questions here :)

It can be tricky sometimes - I usually copy them into an editor and run a few search/replace operations from there.

If it helps -- there is a nice example (I think) for asserting against a list of instruction (or ingredient) lines here: https://github.com/hhursev/recipe-scrapers/blob/03aa8b34648ca7a2c3a21987b52a2d31b67551fb/tests/test_akispetretzikis.py#L53-L69

Reply:
ok, running out of time today, will look into how to properly copy this into the source file tomorrow so that equal assertions work.

Reply:
Sounds good.  No hurry, take your time - thanks for helping out with the fixes.  I'll try to spend more time to review over the next few days.

Reply:
ok so i found this page where you can paste the line broken text and inline it with escaped `\n` characters https://onlinestringtools.com/escape-string

this can then simply be copied into the test. Just need to get an encoding warning dealt with for lecker.de and everything should be fine.